pages/irc.php

```diff
@@ -1,14 +1,22 @@
 <?php
 $title = __("IRC Page");
+if(!$loguserid)
+{
+  Kill("You should be log in for use the IRC.");
+}
+else
+{
+$nick = "insert default guest nick here";

```

Comment:
A good reason to require login would be keeping idiots like eyelyke2shit
out.

No bonus for the shitty English in the Kill() message, though.

On Thu, Mar 5, 2015 at 6:07 PM, Konrad Borowski <notifications@github.com>
wrote:

> In pages/irc.php
> <https://github.com/LifeMushroom/Blargboard-extras/pull/12#discussion_r25881002>
> :
>
> > @@ -1,14 +1,22 @@
> >  <?php
> >  $title = __("IRC Page");
> > +if(!$loguserid)
> > +{
> > +  Kill("You should be log in for use the IRC.");
> > +}
> > +else
> > +{
> > +$nick = "insert default guest nick here";
>
> No. If you require logging in (for absolutely no reason) then this is
> completely stupid.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LifeMushroom/Blargboard-extras/pull/12/files#r25881002>
> .
>